blog/tests/test_models.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for models."""
+from unittest.mock import patch
+from datetime import datetime
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.utils import timezone
+
+from blog.models import Post
+
+
+class ModelPostTest(TestCase):
+    """Main class for testing Post models of this project."""
+
+    def setUp(self):
+        """Prepare data for testing."""
+        self.user = User.objects.create(username='testuser')
+        self.test_post = Post.objects.create(author=self.user, title='Test', text='superText')
+
+    def test_post_rendering(self):
+        """Post is rendered as its title."""
+        self.assertEqual(str(self.test_post), self.test_post.title)
+
+    @patch('django.utils.timezone.now', lambda: datetime(day=1, month=4, year=2016,
```

Comment:
Do you know about mock, patch and why they are needed?

Reply:
Yep